javers-persistence-sql/src/main/java/org/javers/repository/sql/schema/FixedSchemaFactory.java

```diff
@@ -39,6 +39,7 @@
     public static final String SNAPSHOT_COMMIT_FK =    "commit_fk";
     public static final String SNAPSHOT_GLOBAL_ID_FK = "global_id_fk";
     public static final String SNAPSHOT_TYPE =         "type";
+    public static final String VERSION =               "version";
```

Comment:
You are right, there is no elegant way to serialize version in CdoSnapshotState. Looks like we need to come back to your original solution and write `addColumnIfNotExists()` method in `JaversSchemaManager`. We need to take a deep dive into PollyJDBC code. It's very lightweight but simple library ...